testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/JMSMessageDrivenBeanTestCase.java

```diff
@@ -58,7 +59,8 @@
     public static Archive<?> deployment() {
         final Archive<?> deployment = ShrinkWrap.create(JavaArchive.class, "JMSMessageDrivenBeanTestCase.jar")
                 .addClasses(ReplyingMDB.class,JMSMessagingUtil.class)
-                .addClass(CreateQueueSetupTask.class);
+                .addClass(CreateQueueSetupTask.class)
+                .addClass(TimeoutUtil.class);
```

Comment:
The deployment will now need a system property read permission for this test to work with a security manager. MDBTestCase has an example of that setup.